reference-adapter/ifix-department-entity-service/src/main/java/org/egov/validator/DepartmentEntityValidator.java

```diff
@@ -121,4 +115,82 @@ public void validateSearchDepartmentEntity(DepartmentEntitySearchRequest departm
             throw new CustomException(errorMap);
         }
     }
+
+    /**
+     * @param departmentEntityRequest
+     */
+    public void validateUpdateDepartmentEntityRequest(DepartmentEntityRequest departmentEntityRequest) {
+        Map<String, String> errorMap = new HashMap<>();
+
+        if (departmentEntityRequest != null && departmentEntityRequest.getRequestHeader() != null
+                && departmentEntityRequest.getDepartmentEntity() != null) {
+
+            DepartmentEntity departmentEntity = departmentEntityRequest.getDepartmentEntity();
+
+            if (StringUtils.isEmpty(departmentEntity.getId())) {
+                errorMap.put(DepartmentEntityConstant.ID, "Department Entity id is missing in request data");
+            }
+
+            if (StringUtils.isEmpty(departmentEntity.getTenantId())) {
```

Comment:
Why haven't you reused the code of create validator?

Reply:
Both create and update have different mandatory fields. In create we are check every mandatory field by null check but this scenario doesn't fit for update.